src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -317,27 +364,6 @@
     </GenerateCurrentVersion>
   </Target>
 
-  <Target Name="GetVersionSeedDate" Condition="'$(VersionSeedDate)'==''">
-    <ItemGroup>
-      <VersionTargetsFile Include="$(MSBuildThisFileFullPath)" />
-    </ItemGroup>
-      
-    <!-- Windows Exec command requires DOS escaping for the percent sign since it secretly runs the process in a shell instead of calling createprocess. -->
-    <PropertyGroup>
-      <GitCommand Condition="'$(OsEnvironment)'=='Windows_NT'">git show -s --format=^%25%25cd --date=short HEAD</GitCommand>
-      <GitCommand Condition="'$(OsEnvironment)'!='Windows_NT'">git show -s --format=%25cd --date=short HEAD</GitCommand>
-    </PropertyGroup>
-      
-    <Exec Command="$(GitCommand)" StandardOutputImportance="Low" IgnoreExitCode="true" IgnoreStandardErrorWarningFormat="true" ConsoleToMSBuild="true">
-      <Output TaskParameter="ConsoleOutput" PropertyName="VersionSeedDate" />
-      <Output TaskParameter="ExitCode" PropertyName="GitCommandExitCode" />
-    </Exec>
-    <PropertyGroup Condition="'$(GitCommandExitCode)'!='0'">
-      <VersionSeedDate>%(VersionTargetsFile.ModifiedTime)</VersionSeedDate>
-      <VersionSeedSourceComment>VersionSeedDate was produced by getting the timestamp of versioning.targets.</VersionSeedSourceComment>
-    </PropertyGroup>
-  </Target>
-
   <!-- This target will only be executed if BuildVersion.props doesn't exist yet -->
   <Target Name="CreateVersionFileDuringBuild" Condition="'$(SkipVersionGeneration)'!='true' AND '$(ShouldCreateVersionFileDuringBuild)'=='true'" DependsOnTargets="CreateOrUpdateCurrentVersionFile">
```

Comment:
The commit sha in the VersionInfo file isn't being used to invalidate it. The file is "invalidated" when it gets deleted in a clean. I can't see the value in putting the commit sha in the BuildVersion.props file.